frontend/src/components/BookmarkListSidePanel.vue

```diff
@@ -45,6 +48,22 @@ export default {
       store.dispatch("bookmark/deleteBookmark", bookmarkList.value[index]);
     };
 
+    const kbarActions = computed((): Action[] => {
+      const actions = bookmarkList.value.map((item: any) =>
+        createAction({
```

Comment:
nit: I prefer registerAction over createAction to feel like a more extensible component.

Reply:
`createAction` is used only for type inferring. Just like Vue's `defineComponent`.
It comes from the same concept of timc1/kbar.
`useRegisterAction` is the real registration function.
I think we may rename `createAction` to `defineAction` if it is confusing.

Reply:
I see. NM